marketmaker.js

```diff
@@ -18,298 +22,345 @@ const UNISWAP_V3_PROVIDERS = {};
 const PAST_ORDER_LIST = {};
 const FEE_TOKEN_LIST = [];
 let FEE_TOKEN = null;
+let SECRET = {};
 
 let uniswap_error_counter = 0;
 let chainlink_error_counter = 0;
 
 // Load MM config
 let MM_CONFIG;
 if (process.env.MM_CONFIG) {
-    MM_CONFIG = JSON.parse(process.env.MM_CONFIG);
-}
-else {
-    const mmConfigFile = fs.readFileSync("config.json", "utf8");
-    MM_CONFIG = JSON.parse(mmConfigFile);
+  MM_CONFIG = JSON.parse(process.env.MM_CONFIG);
+} else {
+  const mmConfigFile = fs.readFileSync("config.json", "utf8");
+  MM_CONFIG = JSON.parse(mmConfigFile);
 }
 if (MM_CONFIG.feeToken) {
   FEE_TOKEN = MM_CONFIG.feeToken;
 }
 let activePairs = [];
 for (let marketId in MM_CONFIG.pairs) {
-    const pair = MM_CONFIG.pairs[marketId];
-    if (pair.active) {
-        activePairs.push(marketId);
-    }
+  const pair = MM_CONFIG.pairs[marketId];
+  if (pair.active) {
+    activePairs.push(marketId);
+  }
 }
 console.log("ACTIVE PAIRS", activePairs);
 
+if (MM_CONFIG.secretManager.enable) {
+  // Set the AWS Region.
+  const REGION = "us-east-1";
```

Comment:
Why is the REGION hardcoded? Should be configurable for other regions as well.

Reply:
Sorry for that I forgot to move it to config.json after coding. Let me change it.